analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -34,7 +34,7 @@ import org.ossreviewtoolkit.model.yamlMapper
 import org.ossreviewtoolkit.utils.DiskCache
 import org.ossreviewtoolkit.utils.Os
 import org.ossreviewtoolkit.utils.collectMessagesAsString
-import org.ossreviewtoolkit.utils.getUserOrtDirectory
+import org.ossreviewtoolkit.utils.getOrtDataDirectory
```

Comment:
Gradle also has `GRADLE_USER_HOME` for exactly the same purpose we have this variable. That doesn't mean I'm against this change, except if we want to align with Gradle naming, but using `GRADLE_HOME` as example in the commit message is not the best choice.

Reply:
> Gradle also has `GRADLE_USER_HOME` for exactly the same purpose we have this variable.

I know, which is why I had called the variable originally `ORT_USER_HOME`, but I still find that name misleading for the very reason given in the commit message. My intention was not to align with Gradle, but rather mention `GRADLE_HOME` as a negative example. IMO "home" always has the notion of "user home", not "home of the application".

> but using `GRADLE_HOME` as example in the commit message is not the best choice.

Not even as a negative example? What would be a better example in your view?

Reply:
I didn't understand it as a negative example. As we discussed recently that we also want ORT to read configuration files from this directory by default, I think "data dir" is not the best name, but I also have no better idea than "user home" currently.

Reply:
> we also want ORT to read configuration files from this directory

My plan was to introduce a separate "config" directory below the ".ort" directory, and also a separate `ORT_CONFIG_DIR` variable for that. This way, you could also put config into a completely different directory hierarchy than the rest of the data. Does that sound feasible to you?

Reply:
FYI, I've used `JAVA_HOME` as the example now.

Reply:
I would actually have preferred to have the config files directly in the .ort directory, e.g. `.ort/ort.conf` instead of `.ort/config/ort.conf`. This could work if `ORT_CONFIG_DIR` and `ORT_DATA_DIR` both point to `~/.ort` by default, because there is no overlap in filenames.

Reply:
But as we have many configuration files it's maybe better to have a separate config directory, I'm undecided right now.

Reply:
My main reason to put config files into a subdirectory is to be easily able to get / store config from / in a Git repository.

Reply:
You could set `ORT_CONFIG_DIR` then, this would not contradict having `~/.ort` as the default config dir. Anyway, I'll approve this now, we could discuss further details in chat if required.

Reply:
> You could set `ORT_CONFIG_DIR` then

Correct, but this way I do not even have to set `ORT_CONFIG_DIR`, but can create `~./ort/config` by directly cloning a config repo inside `~/.ort`. Let's see how this goes, I'm fine with reorganizing this later again.